data_write/data_write.py

```diff
@@ -56,36 +56,51 @@ def printing(msg):
     "borealis_git_hash" : None, # Identifies the version of Borealis that made this data.
```

Comment:
Example of a git hash, and how to find out what your git hash is should be placed in the documentation I think. (maybe not here, just thinking about this right now).

Reply:
@kkotyk

Reply:
Will add a section to the documentation for data fields.